From 258e54b9bcf96e0535de4f56c3ed30092c8dbf4e Mon Sep 17 00:00:00 2001
From: Karthikeyan Kathirvel <quic_kathirve@quicinc.com>
Date: Thu, 31 Aug 2023 18:10:59 +0530
Subject: [PATCH 8/8] Revert "wifi: ath12k: wmi: remove unused struct
 ath12k_wmi_peer_assoc_arg::peer_macaddr"

This reverts commit 07eec7b45fe6adf291ebd3f4adb8bc3b86aa0d6f.

Signed-off-by: Gautham Kumar Senthilkumaran <quic_gauthamk@quicinc.com>
Signed-off-by: Karthikeyan Kathirvel <quic_kathirve@quicinc.com>
---
 drivers/net/wireless/ath/ath12k/wmi.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/net/wireless/ath/ath12k/wmi.h b/drivers/net/wireless/ath/ath12k/wmi.h
index c7eca765fa70..b9c2b5bafad3 100644
--- a/drivers/net/wireless/ath/ath12k/wmi.h
+++ b/drivers/net/wireless/ath/ath12k/wmi.h
@@ -3484,6 +3484,7 @@ struct wmi_rate_set_arg {
 };
 
 struct ath12k_wmi_peer_assoc_arg {
+	struct wmi_mac_addr peer_macaddr;
 	u32 vdev_id;
 	u32 peer_new_assoc;
 	u32 peer_associd;
-- 
2.38.0

